src/sparsezoo/api/graphql.py

```diff
@@ -27,8 +27,8 @@ class GraphQLAPI:
     def fetch(
         self,
         operation_body: str,
-        arguments: Optional[Dict[str, str]] = None,
-        fields: Optional[List[str]] = None,
+        arguments: Optional[Dict[str, Any]] = None,
+        fields: Optional[Union[List[str], Dict[str, Union[Dict, None]]]] = None,
```

Comment:
Can you get away with `Optional[Dict]` and avoid the need for one of the `Union`?
```suggestion
        fields: Optional[Union[List[str], Dict[str, Optional[Dict]]]] = None,
```